sites/shared/utils.mjs

```diff
@@ -187,8 +187,9 @@ export const optionsMenuStructure = (options) => {
   // Fixme: One day we should sort this based on the translation
   for (const option of orderBy(sorted, ['menu', 'name'], ['asc'])) {
     if (typeof option === 'object') {
+      const isAConstant = Object.keys(option).length === 1
       if (option.menu) set(menu, `${option.menu}.${option.name}`, optionType(option))
-      else if (typeof option.menu === 'undefined') {
+      else if (typeof option.menu === 'undefined' && !isAConstant) {
```

Comment:
I think this check, and actually also the menu check, would work better in the previous loop on line 182. We're manipulating the options there anyway, may as well only include ones that should actually go in the menu. That way instead of counting keys, you can do a type check on the options. Then this loop will just do the business of line 192

Reply:
@BenJamesBen please change line 183 to:

```js
if (typeof option === 'object') sorted[name] = { ...option, name }
```

The current behaviour is a bug, and this proposed change treats the symptom, rather than the cause.

Static options are a scalar, and a scalar can't be spread (`...5` is not valid JS).
The reason it doesn't throw an error is because it's in an object literal (`{ ...option}`). However, if you were to use `console.log(...option)` it would throw an error as, once again, you can't spread a scalar.

Because it's in the object literal, this error is not thrown, but it's still a bug and that's what should be addressed.

So, if you feel up for it, you could revert this change and instead update line 183 as shown above.